src/tools/miri/src/shims/intrinsics/simd.rs

```diff
@@ -75,8 +75,14 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
                             };
                             let op = op.to_scalar();
                             match float_ty {
+                                #[cfg(bootstrap)]
+                                FloatTy::F16 | FloatTy::F128 => unimplemented!(),
```

Comment:
Why does this need `cfg(bootstrap)`? It doesn't involve any host floats I think?